### DISCUSSION TITLE
Not an issue , just a query regarding Lora Ra01 module

### DISCUSSION THREAD

**Discussion by anandspathak** [jgromes/RadioLib#380] [Q&A] [answered]
Dear Sir,
I have Lora Ra-01 module,it does not have Dio1 pin only Dio0 pin is there
![IMG_20211003_024921](https://user-images.githubusercontent.com/43384018/135732059-2aea6944-6f5b-49b5-a96d-537b913a1002.jpg)
![IMG_20211003_024740](https://user-images.githubusercontent.com/43384018/135732061-a3ad1a31-9560-46a2-a0d3-9a386c4fd5f8.jpg)
,
1). Can you tell me  constructor for this module?
2). Can your Ping Pong example work with this 
       module?? (Been using lora sx1278 blue pcb type 
       which works flawlessly with your Ping Pong 
       example after slight modification).
Thanks,
Regards
Anand Pathak
India

**Comment by anandspathak** [comment]
Also which is better among these two modules
SI4432  vs Lora sx1278 in terms of range???

**Comment by jgromes** [comment, marked as answer]
> Can you tell me constructor for this module?

You can use `RADIOLIB_NC` to mark a pin as "not connected", so the constructor will be 

```c++
SX1278 radio = new Module(10, 2, 9, RADIOLIB_NC);
```

However, you will not be able to use some features, like CAD or receiver timeout detection (so you will have to use interrupt reception).

> Can your Ping Pong example work with this module??

I don't see why not. What modifications did you do?

> Also which is better among these two modules

SX1278 in LoRa mode will get better range, but will achieve much lower datarates.

**Reply by anandspathak** [reply]
I just changed radio name from lora sx1268 to sx1278 and it worked good...thanks for your help and this awesome library... It works well for me...,,

**Comment by anandspathak** [comment]
Thank you very much,
